ql/src/test/results/clientpositive/llap/dynamic_semijoin_reduction_3.q.out

```diff
@@ -261,31 +260,21 @@ STAGE PLANS:
                   Statistics: Num rows: 1 Data size: 4 Basic stats: COMPLETE Column stats: NONE
                   Group By Operator
                     aggregations: compute_stats(a, 'hll'), compute_stats(b, 'hll')
-                    minReductionHashAggr: 0.99
-                    mode: hash
+                    mode: complete
```

Comment:
Same as above... complete here would imply a single instance to write data to the table, which could slow down execution. What is the root cause for this?

Reply:
Enabling parallelism when inserting eliminates this change.